parser/src/lexer.rs

```diff
@@ -625,7 +625,10 @@ where
                 }
                 Some('\n' | '\r') => {
                     // Empty line!
+                    let tok_start = self.get_pos();
```

Comment:
Unrelated to your changes. I think this emits two newlines when using \r\n instead of one.

Reply:
I think it might actually work correctly? I think `next_char` already does the advancement:

```rust
// Helper function to go to the next character coming up.
fn next_char(&mut self) -> Option<char> {
    let mut c = self.window[0];
    self.window.slide();
    match c {
        Some('\r') => {
            if self.window[0] == Some('\n') {
                self.location += TextSize::from(1);
                self.window.slide();
            }

            self.location += TextSize::from(1);
            c = Some('\n');
        }
        #[allow(unused_variables)]
        Some(c) => {
            self.location += c.text_len();
        }
        _ => {}
    }
    c
}
```

Reply:
ah, I had forgotten about that :smile: